app/controllers/api/v1/recordings_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class RecordingsController < ApplicationController
+      # GET /api/v1/recordings.json
+      # Returns: { data: Array[serializable objects(recordings)] , errors: Array[String] }
+      # Does: Returns all the Recordings from all the current user's rooms
+      def index
+        # Gets the id's of the rooms that belong to the current user
+        room_ids = Room.where(user_id: current_user&.id).map(&:id)
+        # Gets the recordings that belong to current user
+        recordings = Recording.where(room_id: room_ids)
```

Comment:
If you add a `has_many :recordings through: :rooms` to `User`, you can just do `current_user.recordings` here